proposals/TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+# My New Proposal
+>  Stage 0
+
+Below is simply a suggested template for drafting a proposal, it will not fit all potential proposals.
+
+## Champion
+
+Who is the the stakeholder of this initiative?
+
+## Description
+
+A rough description of the program idea or policy
+
+## Required Resources
+
+What resources would be needed to execute it?
+
+## Who would be responsible?
+
+Which committee would be overseeing or responsible for executing this proposal
+
+## How would success be measured?
+
+Please include potential metrics that can be used to measure success of the proposal.
+
+## Why this proposal is important
```

Comment:
Nit: Question mark at the end.

Reply:
As it is currently written this is a statement not a questions
`Why this proposal is important` vs `Why is this proposal important`